wemake_python_styleguide/violations/complexity.py

```diff
@@ -678,6 +678,11 @@ class TooManyConditionsViolation(ASTViolation):
         creating new variables or creating nested ``if`` statements.
         Both of these actions will trigger other complexity checks.
 
+    Configuration:
+        This rule is configurable with ``--max-conditions``.
+        Default:
+        :str:`wemake_python_styleguide.options.defaults.MAX_CONDITIONS`
+
```

Comment:
Add `.. versionchanged:: 1.4.0` with a description that we made it configurable.